scripts/import/import_clinvar_xml

```diff
@@ -85,8 +85,8 @@ my $dbh = $dba->dbc;
 
 my $variation_adaptor     = $dba->get_VariationAdaptor('human', 'variation', );
 my $var_feat_adaptor      = $dba->get_VariationFeatureAdaptor('human', 'variation', );
-my $allele_adaptor        = $reg->get_adaptor('homo_sapiens', 'variation', 'allele');
 my $structvar_adaptor     = $dba->get_StructuralVariationAdaptor('human', 'variation', );
+my $aa                    = $dba->get_AttributeAdaptor();
```

Comment:
Suggest renaming variable to $attrib_adaptor. $aa sometimes uses for ancestral allele in other code.